doc/release/1.13.0-notes.rst

```diff
@@ -385,7 +397,7 @@ offset into the file. This is a behaviour change only for offsets
 greater than ``mmap.ALLOCATIONGRANULARITY``.
 
 ``np.real`` and ``np.imag`` return scalars for scalar inputs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Well, one of them has to go first ;) Easy fix on line if homu complains, but now I'm actually curious if it will cause a merge problem.